src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -30,7 +29,7 @@ public ProductDto getProduct(@PathVariable Long productId) throws ProductNotFoun
     }
 
     @PostMapping(value = "createProduct", consumes = MediaType.APPLICATION_JSON_VALUE)
-    public void createProduct(@RequestBody CreateProductDto createProduct) {
+    public void createProduct(@RequestBody ProductDto createProduct) {
```

Comment:
wyjątkowo akceptuję, generalnie jak już takie zmiany są robione w ramach jakiegoś zadania, to powinien to być całkowicie osobny commit